README.md

```diff
@@ -133,6 +133,15 @@ Currently `pyproject.toml` is supported.
     version-file: "my-path/to/pyproject.toml"
 ```
 
+And `requirements.txt` is supported.
```

Comment:
Lets combine this instead:

You can specify a file to read the version from.
Currently the `pyproject.toml` or `requirements.txt` format is supported.
```yaml
- name: Install a version from a specified version file
  uses: astral-sh/ruff-action@v3
  with:
    version-file: "my-path/to/pyproject.toml-or-requirements.txt"
```